src/tools/scripts/fixDateDeNaissanceStringToDate.js

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env node
+'use strict';
+
+const { execute } = require('../utils');
+const dayjs = require('dayjs');
+
+execute(__filename, async ({ logger, db, exit }) => {
+  let countCorrection = 0;
+  const promises = [];
+  try {
+    const conseillers = await db.collection('conseillers').find(
+      { dateDeNaissance: { $type: 'string' } }
+    ).toArray();
+    logger.info(`${conseillers.length} conseillers à corriger`);
+    conseillers.forEach(conseiller => {
+      promises.push(new Promise(async resolve => {
+        let query = { $unset: { dateDeNaissance: '' } };
+        let queryMER = { $unset: { 'conseillerObj.dateDeNaissance': '' } };
+        if (dayjs(conseiller.dateDeNaissance).isValid()) {
```

Comment:
Si c'est isValid ca voudrait dire que le set ne sert pas non ?

Reply:
ca réécraserait ce qu 'il y a déjà ?

Reply:
je me suis poser la même question mais si ça servira car ça sera de type Date aux lieux d'un String

Reply:
Ca mais un string en date

Reply:
ok par contre ca va tout réupdate du coup - faudrait pas un check is type string pour maj ?